wikimedia/wikipedia-ios
# [YiR] Display profile button badge and notification badge

**Phabricator:** https://phabricator.wikimedia.org/T376313 and https://phabricator.wikimedia.org/T376312

### Notes
* Updates the badge state based on the user having an YiR report available and has not seen its first slide

### Test Steps
1. Fresh install the app
2. Set region to Italy or Mexico on the Device
3. Go through 7+ articles
4. Turn on dev settings toggle
5. Background the app
6. When foregrounding, you should see the badge on the explore feed and article profile buttons
7. You should see the badge on the YiR item
8. Click on it, view the first slide
9. Go back to the profile view. The badge should be gone
10. Test with notifications badge on - if you have unread notifications, you should still see the profile button badge
